username_api.py

```diff
@@ -86,6 +85,20 @@ def check_username(website, username):
 			'possible': possible,
 		}
 
+	if website == 'opensuse':
+		res = r.get(url)
+		if bytes('Sorry, we could not find the specified profile', encoding='utf-8') in res.content:
```

Comment:
Not sure about this, but is there a way to merge this with the `[pinterest, gitlab]` case? Maybe have some string to match if the code = 200? Then it can be put in a dict or something to avoid repeated logic..

Reply:
Yep, I see the text "About me" only on valid profiles. Let's match that for code == 200 and merge this [pinterest, gitlab] case.

Reply:
@biscuitsnake do you understand the approach here?

Reply:
^ you can push this ``phrase`` data out to websites.yml for the websites to which this rule applies

Reply:
for merging with pinterest and gitlab should I use a dict or just use if, elif

Reply:
@manu-chroma I don't know what you mean by pushing out the phrase data to another file

Reply:
Also, are you sure it's ok to keep sites that redirect and sites that don't together?
It might be better to keep them apart for future sites